requirements/base.txt

```diff
@@ -36,7 +36,7 @@ pandas==2.3.2
 plotly==6.3.0
 # Upgrade to psycopg>=3 once ddtrace supports it.
 # See https://github.com/DataDog/dd-trace-py/issues/4393
-psycopg2-binary==2.9.7
+psycopg2-binary==2.9.10
```

Comment:
The comment above says we should upgrade to psycopg>=3, currently 3.2.10, once ddtrace isn't a blocker. And we have removed ddtrace.